src/acceptance/administrator/components/com_menu/MenuItemCest.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * @package     Joomla.Tests
+ * @subpackage  Acceptance.tests
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+use Step\Acceptance\Administrator\MenuItemStep;
+use Page\Acceptance\Administrator\MenuItemListPage;
+
+/**
+ * Menu Item class
+ *
+ * @category  Menu_Item
+ * @package   Administratorcomponentscom_Menu
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
+ * @link      ArticleMenuCest
+ * @since     4.0.0
+ */
+class MenuItemCest
+{
+    /**
+     * Variables Initialised
+     *
+     * menuItemTitle	string
+     * menuItemAlias	string
+     */
+    public function __construct()
+    {
+        $this->menuItemName = 'Joomla Menu Item';
+        $this->menuItemAlias = 'Joomla Menu Item Alias';
+    }
+    /**
+     * Creates a menu item with the Joomla menu manager
+     *
+     * @param \AcceptanceTester $I
+     *
+     * @throws \Exception
+     * @since  4.0.0
+     * @return void
+     */
+    public function menuItem(\AcceptanceTester $I)
```

Comment:
I think what you should be doing here is this, inject the object for the step class you created instead of injecting AcceptanceTester 
`public function menuItem(MenuItemStep $I)`